packages/repository/src/__tests__/fixtures/mixins/category-property-mixin.ts

```diff
@@ -6,20 +6,26 @@
 import {Constructor} from '@loopback/context';
 import {Entity, property} from '../../..';
 
+/**
+ * A mixin factory to add `category` property
+ *
+ * @param superClass - Base Class
+ * @typeParam T - Model class
+ */
 export function AddCategoryPropertyMixin<T extends Constructor<Entity>>(
   superClass: T,
 ) {
   class MixedModel extends superClass {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    constructor(...args: any[]) {
+      super(...args);
+    }
```

Comment:
-1

I believe our convention is to list properties first, constructor second, other methods third.